### PR TITLE
Set initializer for Material color

### PR DESCRIPTION
**Pull request by elijahsgh**
https://github.com/mrdoob/three.js/issues/7198#issuecomment-141924775

Assuming all material had colors.  Color is now initialized properly.

### CODE REVIEW

**Comment by threejsworker** [comment]
The examples of this pullrequest are now built and visible in threejsworker. To view them, go to the following link: 

 http://threejsworker.com/viewpullrequest.html#7334

File: examples/js/loaders/AMFLoader.js [hunk 1]
```diff
@@ -206,6 +206,7 @@ THREE.AMFLoader.prototype = {
 
 				} else if ( matColor.nodeName === 'a' ) {
 
+					color.a = matColor.textContent;
```
Comment (mrdoob) [inline examples/js/loaders/AMFLoader.js]: What's the difference between `a` and `opacity`?

Comment (tamarintech) [inline examples/js/loaders/AMFLoader.js]: 6 letters? :monkey: 

Removed. :)  There was also a typo in materialid that had broken materials.  Updated.

**Comment by mrdoob** [comment]
Sweet!

**Pull request merged**